packages/messaging/src/extension.test.ts

```diff
@@ -23,7 +23,7 @@ describe('Messaging Wrapper', () => {
     vi.restoreAllMocks();
   });
 
-  it('should send and recieve messages', async () => {
+  it('should send and receive messages', async () => {
```

Comment:
Oh yeah, I also fixed a ton of typos lol. I could not spell "receive" correctly :laughing: